.github/workflows/ci-pr.yml

```diff
@@ -5,6 +5,7 @@
 
 name: CI
 on:
+  workflow_dispatch:
```

Comment:
What is this for?

Reply:
It gives you the possibility to trigger the workflow from GitHub's UI.
See `ci-main`, which already has it:
<img width="1288" alt="image" src="https://user-images.githubusercontent.com/20745048/194268546-3345b511-d561-4916-b3f6-48c556aac5de.png">

Reply:
But why would you need this for `ci-pr.yml`? It runs every time a PR is updated.

And if it is run via workflow dispatch, how does it get values needed by the PR bot:
```
      issue_number: ${{ github.event.number }}
      base_revision: ${{ github.event.pull_request.base.sha }}
      head_revision: ${{ github.event.pull_request.head.sha }}
```